package.json

```diff
@@ -207,6 +208,8 @@
     "@types/enzyme": "3.10.0",
     "@types/enzyme-adapter-react-16": "1.0.2",
     "@types/express": "^4.17.2",
+    "@types/fetch-mock": "^7.3.2",
+    "@types/hoist-non-react-statics": "^3.3.1",
```

Comment:
If you require 3.3.1, please also update the hoist-non-react-statistics to the ^3.3.1 please, lets not have a discrepancy of types and installed version.

Reply:
Good catch. I don't actually don't require this types. I'm confused as how this happened, but I suspect the editor did something on behalf of me and I overlooked it. Removing it. Thanks.